recipes/parfive/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "parfive" %}
+{% set version = "0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0ec7a208f9574bcd855d9ca02dbc0cb4ccf67d49b7fb6b44b908bcf11e2cf893
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools_scm
+  run:
+    - python
+    - setuptools
```

Comment:
Do you need ``setuptools`` to run this project?

Reply:
I did a quick look at this project and didn't see ``setuptools`` as a requirement to run it. I may be wrong, but could you please double check that?

Reply:
We use `pkg_resources` to get the version number [here](https://github.com/Cadair/parfive/blob/master/parfive/__init__.py#L13) as suggested by setuptools_scm, so I think that means we need setuptools?

Reply:
In this case you ``setuptools`` is a dependency, thanks for double checking it! :)